init-tools.sh

```diff
@@ -93,14 +93,12 @@ if [ ! -e $__DOTNET_PATH ]; then
                 if [ -e /etc/os-release ]; then
                     source /etc/os-release
                     if [[ $ID == "alpine" ]]; then
-                        # remove the last version digit
-                        VERSION_ID=${VERSION_ID%.*}
-                    __PKG_RID=alpine.$VERSION_ID
+                        __PKG_RID=linux-musl
```

Comment:
We are going to have to bump the CLI version right?  We don't currently have a linux-musl CLI for `v2.1.2`.  Or did you make a "bootstrap" CLI for it?

Reply:
I've not yet but I plan to.

Reply:
Which do you plan to do?  I assume bump the CLI version....

Reply:
No what I did was copy the alpine cli in the 2.1.2 and named in linux-musl.